subsys/bluetooth/controller/ll_sw/ull_iso.c

```diff
@@ -180,12 +214,15 @@ uint8_t ll_setup_iso_path(uint16_t handle, uint8_t path_dir, uint8_t path_id,
 	}
 
 	if (path_id == BT_HCI_DATAPATH_ID_HCI) {
-		/* Not vendor specific, thus alloc and emit functions known */
+		/* Set up HCI data path */
 		err = isoal_sink_create(&sink_hdl, handle, burst_number, sdu_interval,
 					cig->iso_interval, sink_sdu_alloc_hci,
 					sink_sdu_emit_hci, sink_sdu_write_hci);
 	} else {
-		/* TBD call vendor specific function to set up ISO path */
+		/* Set up vendor specific data path */
+		err = isoal_sink_create(&sink_hdl, handle, burst_number, sdu_interval,
+					cig->iso_interval, sink_sdu_alloc_vs,
+					sink_sdu_emit_vs, sink_sdu_write_vs);
```

Comment:
I am wondering whether this call should actually go to a vendor-specific function instead of having the alloc/emit/write functions vendor specific. That would allow using different alloc/emit/write functions for different data path IDs. For example, a device could use different data path IDs for rendering on a built-in speaker or on a digital output.